pkg/datastore/database.go

```diff
@@ -534,20 +558,11 @@ func dbReadNotifier(notifierName string, fetchRecipients bool) (*Notifier, error
 		return nil, err
 	}
 
-	duration, err :=
-		time.ParseDuration(strings.ToLower(strings.Split(readNotifier.Interval, "T")[1]))
-	if err != nil {
-		log.Fatal(err)
-		return nil, err
-	}
-	//log.Tracef("%#v\n", duration.String())
-	readNotifier.Interval = duration.String()
-
 	if fetchRecipients {
```

Comment:
always fetch the recipients

Reply:
solved by [1095c0d](https://github.com/JM-Lemmi/ical-relay/pull/233/commits/1095c0d9257d368ca42ae6ff07330913d3f70f48)